tooling/minimize/src/function.rs

```diff
@@ -107,25 +107,87 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
             self.locals.insert(*local_name, ty);
         }
 
+        let mut args = List::default();
+        let mut init_statements: Vec<Statement> = Vec::new();
+        let mut spread_assignments = Vec::new();
```

Comment:
`spread_assignments` is only used inside the "then" branch, please move it there.